src/compas_rhino/geometry/brep/face.py

```diff
@@ -0,0 +1,41 @@
+from compas.data import Data
+from compas_rhino.geometry import RhinoNurbsSurface
+
+from .loop import RhinoBRepLoop
+
+
+class RhinoBRepFace(Data):
+
+    TOLERANCE = 1e-6
```

Comment:
need a better solution for this.
also just in general, we need a formal location to store `compas`-wide configuration options...

Reply:
@gonzalocasas mentioned once these can be handled with some global COMPAS precision system. I'll look into it.

Reply:
well, i wouldn't necessarily call it a system, but there is indeed a precision setting `compas.PRECISION` :) 
a more robust version of this that also allows for other settings would be a good thing...